config/application.rb

```diff
@@ -33,7 +33,8 @@ class Application < Rails::Application
       event.payload.except(:params, :headers)
     end
 
-    config.middleware.insert_before 0, Rack::HeadersFilter
```

Comment:
It looks like the `rack-header-filter` gem [let's you call this with a blacklist](https://github.com/helle/rack-header-filter/blob/master/lib/rack/header_filter.rb#L5). Why not do that here?

Reply:
Oh thanks, I'd missed that gem entirely, will update

Reply:
Ok, so that gem deletes outgoing headers, not incoming headers, I don't think we can use it here

Reply:
Frown town. Oh well

Reply:
Extremely frown town 😿